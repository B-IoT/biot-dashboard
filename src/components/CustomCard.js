import { Card as MaterialCard } from "@material-ui/core";
import CardContent from "@material-ui/core/CardContent";

export default function CustomCard(props) {
  return (
    <MaterialCard className={props.className} style={props.style}>
      <CardContent>{props.children}</CardContent>
    </MaterialCard>
  );
}
