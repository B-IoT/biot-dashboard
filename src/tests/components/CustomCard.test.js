import renderer from "react-test-renderer";
import CustomCard from "../../components/CustomCard";

it("renders without crashing with custom props", () => {
  const props = {
    className: test,
    style: { color: "#ffff" },
    children: 1,
  };
  const rendered = renderer.create(<CustomCard props={props} />).toJSON();
  expect(rendered).toMatchSnapshot();
});
