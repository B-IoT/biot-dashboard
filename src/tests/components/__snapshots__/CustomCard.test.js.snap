// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing with custom props 1`] = `
<div
  className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  />
</div>
`;
